linkedin.py

```diff
@@ -454,23 +454,24 @@ def handleMultiplePages(self, jobPage, jobProperties: models.Job, jobCounter: mo
             jobCounter = self.cannotApply(jobPage, jobProperties, jobCounter)
             return jobCounter
         
-        percentageElement = self.driver.find_element(By.XPATH, constants.multiplePagePercentageXPATH)
+        percentageElement = WebDriverWait(self.driver, 10).until(
```

Comment:
Use self.exists() instead of waiting

Reply:
Fixed, now instead of the wait, the new logic uses the `exists` function.